wiki/Help_centre/Upgrading_to_lazer/ro.md

```diff
@@ -0,0 +1,312 @@
+
+În final, spre deosebire de iterația anterioară, ne asigurăm că baza codului este prezervată cu atenție și menținută cumsecade pentru viitor. Am pus bazele acestea pentru a putea implementa funcții noi mai repede, precum componente UI, modalități diferite de skinning, mai multe sisteme multiplayer și nu în cele din urmă posibilitatea să joci beatmap-urile tale favorite pe diferite moduri de joc!
+
+#### Ce urmează?
+
+Avem un teanc de funcții cerute de utilizatori pe care le vom implementa în curând. Pentru cei care au venit mai târziu, și nu știu cum facem noi treaba pe aici la osu!, fi pregătit să fi lăsat mască.
+
+#### Cum pot să ajung la dosarul meu de melodii?
+
+Nu există un dosar de melodii în lazer! Asta ne oferă posibilitatea să eliminăm funcția de reîmprospătare a beatmap-urilor cu `F5` și să reducem spațiul de stocare folosit de melodii cu 20-40%. Poți să citești mai mult despre [cum lazer stochează fișierele](/wiki/Client/Release_stream/Lazer/File_storage).
+
+Dacă trebuie să modifici ceva la un beatmap, te rugăm să folosești editorul. În viitor vom adăuga o funcție în editor pentru a putea vedea temporar dosarul cu beatmap-uri pentru a putea fi modificat. Asta îți va permite să folosești programe externe pe un beatmap.
+
+#### Acuma că „osu!direct” este disponibil pentru toți jucătorii, ce se întâmplă cu cei care au supporter? Vor primii beneficii noi?
+
+Niște filtre în osu!direct sunt disponibile doar pentru supporteri.
+
+Pe lângă asta, suporterii mai au niște funcții deja implementate.
+
+- Supporterii pot crea playlist-uri care durează mai mult.
+- Supporterii pot activa storyboard în meniul principal.
+
+Suntem interesați să adăugăm mai multe beneficii în viitor, dar momentan lucrăm să aducem mai multe funcții din stable în lazer, așa că te rugăm să iei în considerare cumpărarea tag-ului de supporter.... ca un semn de susținere a jocului!
+
+#### Dacă folosesc coduri / cheaturi, voi primii ban?
+
+Dap.
+
+#### Dacă găsesc pe cineva care joacă cu cod / cheat pe lazer, cum ar trebuii să îl raportez?
+
+La fel cum făceai și înainte.
+
+#### Unde sunt microtranzacțiile?
+
+Cred că ai greșit jocul.
+
+## Note
+
+[^wine]: Folosind Wine.
+[^compatibility-mode]: DirectX cu modul de compatibilitate.
+[^dll]: Manual folosind fișiere `.dll`
+[^share-files]: Beatmap-urile și skin-urile vor impărții fișierele, și va salva din spațiul de stocare.
+[^gameplay-only]: Doar gameplay.
+[^online]: Prin vizualizare online.
+[^normalisation]: Aduce culorile personalizate combo din beatmap la același nivel de luminozitate.
+[^hold-for-hud]: Ține apăsat `Ctrl` pentru a vedea HUD-ul momentan atunci când este ascuns.
+[^offset-calibration-stable]: Ajustabil manual prin taste.
+[^offset-calibration-lazer]: Când reîncerci un beatmap, poți calibra offsetul bazat pe ultima ta încercare.
+[^can-disable]: Poate fi dezactivat.
+[^note-lock]: Încă există, dar n-ar trebuii să încurce.
```

Comment:
```suggestion
[^note-lock]: Încă există, dar n-ar trebuii să încurce.
```
```suggestion
[^note-lock]: Încă există, dar nu ar trebui să încurce.
```

See: https://www.cumzic.ro/ar-trebui-sau-ar-trebuii/ (you make this mistake throughout the entire document)